torch/nn/modules/flatten.py

```diff
@@ -57,8 +58,10 @@ class Unflatten(Module):
       (tuple of `(name, size)` tuples) for `NamedTensor` input.
 
     Shape:
-        - Input: :math:`(N, *dims)`
-        - Output: :math:`(N, C_{\text{out}}, H_{\text{out}}, W_{\text{out}})`
+        - Input: :math:`(*, S_{dim}, *)`. Input can be any shape, where :math:`S_{dim}`
```

Comment:
nit: I think it reads better without the "Input can be any shape" part now but could go either way